src/telemetry.ts

```diff
@@ -1059,11 +1056,8 @@ export class IEventNamePropertyMapping {
         feature: ['InteractiveWindow', 'Notebook'],
         tags: ['Cell Execution'],
         source: 'N/A',
-        properties: commonClassificationForResourceSpecificTelemetryProperties().properties,
-        measures: {
```

Comment:
These telemetry events incorrectly defined, we didn't need these additional properties